Pod/Classes/UITextField+Extensions.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  UITextField+Extensions.swift
+//  Pods
+//
+//  Created by Christopher Jones on 3/24/16.
+//
+//
+
+import UIKit
+
+internal extension UITextField {
+
+    internal static var emptyTextFieldCharacter: String {
+        return "\u{202F}"
```

Comment:
Makes sense! I added it to [my pull request](https://github.com/prolificinteractive/Caishen/pull/13) for the `refactor` branch.